pytest.ini

```diff
@@ -1,2 +1,2 @@
 [pytest]
-addopts = --mpl --cov=shap tests --cov-report=term-missing
\ No newline at end of file
+addopts = --mpl --cov=shap --cov-report=term-missing --durations=20
```

Comment:
Is this causing timeout issues in the CI? Don't think we want to skip tests by default in the pytest.ini Maybe that setting could be applied by a user if the tests are taking too long, or in the offending CI config file if the CI is failing

Reply:
This only prints the durations of the 20 longest tests.

Reply:
Ah my bad, good shout!